addons/binding/org.openhab.binding.systeminfo/ESH-INF/thing/channels.xml

```diff
@@ -170,6 +173,14 @@
 		<config-description-ref uri="systeminfo:channels:highpriority" />
 	</channel-type>
 
+	<channel-type id="usedPercent">
+		<item-type>Number</item-type>
+		<label>Used (%)</label>
+		<description>Used size in percents</description>
```

Comment:
To late but I think the most correct English would actually be 'Used size as a percentage'